modular-docs-manual/content/topics/creating_procedure_modules.adoc

```diff
@@ -1,13 +1,40 @@
-[id="creating-procedure-modules"]
-= Creating Procedure Modules
+[id="proc-creating-procedure-modules_{context}"]
+= Procedure modules
+Procedure modules explain how to do something. A procedure module contains numbered, step-by-step instructions to help the user accomplish a single task. Sometimes those tasks include sub steps. Procedure modules must include a title, a brief introduction, and one or more steps in the form of imperative statements. Procedure modules can also contain prerequisites, verification steps, and additional resources or next steps.
 
-This section explains what a procedure module is and provides recommended practices for writing procedure modules.
+.Schema of a procedure module
+image::procedure-diagram.png[]
 
-include::module_definition-procedure.adoc[leveloffset=+1]
+.Procedure Title
+The title of a procedure module is a verb or a gerund phrase, for example *Deploy OpenShift Container Platform* or *Deploying OpenShift Container Platform*.
```

Comment:
Is FCC now permitting non-gerund verb-phrase titles for procedures? I thought the instruction has always been to use only gerunds.